oesukam/lit-blogpost
# #163520012 Implement delete a post feature

#### What does this PR do?
Add delete a post endpoint

#### Description of Task to be completed?
- Improve on the db migration structure
- Create sequelize seed for users and posts
- Add npm script for sequelize
- Correct jest test

#### How should this be manually tested?
- Clone the repo
- Run: `npm run install`
- Run: `npm run db:migrate:seed`
- Run: `npm run test` to check on delete a post test

#### What are the relevant pivotal tracker stories?
`#163520012`
